tests/composer/auto_complete/function_auto_complete.test.ts

```diff
@@ -0,0 +1,18 @@
+import { ComposerStore } from "../../../src/components/composer/composer/composer_store";
+import { setCellContent } from "../../test_helpers/commands_helpers";
+import { makeStore } from "../../test_helpers/stores";
+
+describe("Function auto complete", () => {
+  test("start with exact match, but with other proposals", () => {
```

Comment:
A specific reason to create a new file for this test rather than putting it with the others in `autocomplete_dropdown_component.test` ?